### ISSUE TITLE
After unfork is completed, show next steps

### ISSUE THREAD

**Issue by marccampbell** [replicatedhq/unfork#11]
We should show the next steps after successfully unforking.

This are:

```
Your unforked Chart is available at ~/path/name

You can install the same version with:
kubectl apply -k ~/path/name/overlays/downstreams/unforked

You can update to the latest version of name with:
kots pull helm://name 
```

**Comment by laverya** [comment]
@marccampbell should unfork continue running after an individual helm chart is unforked? That is the current behavior.

Or do you want this as a dialog after each unfork action?

**Comment by laverya** [comment]
I've added it after each unfork action for now.

**Comment by laverya** [comment]
Fixed by pr #19.
